WalletWasabi.Tests/RegressionTests/CoinJoinTests.cs

```diff
@@ -721,18 +721,18 @@ public async Task CcjEqualInputTestsAsync()
 			// We have to 4 participant so, this data structure is for keeping track of
 			// important data for each of the participants in the coinjoin session.
 			var participants = new List<(AliceClient4 aliceClient,
-										 List<(Requester requester, BitcoinWitPubKeyAddress outputAddress, BlindedOutputWithNonceIndex blindedScript)> outouts,
+										 List<(Requester requester, BitcoinAddress outputAddress, BlindedOutputWithNonceIndex blindedScript)> outouts,
```

Comment:
Is the identifier intentionally `outouts`?

Reply:
Fixed!